### ISSUE TITLE
project source file got filtered out

### ISSUE THREAD

**Issue by dashessmith** [swaggo/swag#1579]
**Describe the bug**
if project dir name start withs "runtime.GOROOT()", swag filtered all source file out
like project name /root/goxxxxx, and runtime.GOROOT() = /root/go

**To Reproduce**
1. make a project at dir which path starts with "runtime.GOROOT()"
2. write some correct swag comments in any subdir
3. swag init -d xxx,xxx

**Expected behavior**
swag do recognize these file

**Screenshots**
![屏幕截图 2023-05-05 161048](https://user-images.githubusercontent.com/75690316/236410011-57c1261b-4e94-4b56-b08c-8e4563739f1a.png)

**Your swag version**
1.8.12

**Your go version**
 go1.20.4 linux/amd64

**Desktop (please complete the following information):**
no need

**Additional context**
may be add a slash at the end, to check whether it's a in a goroot dir

**Comment by Nerzal** [comment]
TL;DR : This currently is the expected behavior

In general i wouldn't place go projects into the GOROOT folder. Only your gosdk should live there. 

Aren't you deleting all your projects with every go update (atleast on linux)?
The default GOROOT in linux is something like `/usr/local/go` which i always delete before unpacking a new go version into that directory.

Example from the install instructions: 

> rm -rf /usr/local/go && tar -C /usr/local -xzf go1.20.4.linux-amd64.tar.gz

Source: [https://go.dev/doc/install](https://go.dev/doc/install)
